lib/traject/penn_near_east_config.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+# Numismatics CSV Mapping Configuration
+
+require_relative 'csv_reader'
+require_relative 'dlme_json_resource_writer'
+require_relative 'macros/dlme'
+require_relative 'macros/csv'
+extend Macros::DLME
+extend Macros::Csv
+
+settings do
+  provide 'writer_class_name', 'DlmeJsonResourceWriter'
+  provide 'reader_class_name', 'CsvReader'
+end
+
+to_field 'agg_provider', provider
+
+# MET Museum
+to_field 'id', normalize_prefixed_id('emuIRN')
+
+to_field 'cho_provenance', column('accession_credit_line')
+to_field 'cho_creator', column('creator')
+to_field 'cho_coverage', column('culture', split: '|')
+to_field 'cho_coverage', column('culture_area')
+to_field 'agg_data_provider', normalize_penn_egyptian_provider
+to_field 'cho_date', column('date_made')
+to_field 'cho_date', column('date_made_early')
+to_field 'cho_date', column('date_made_late')
+to_field 'cho_description', column('description')
+to_field 'cho_identifier', column('emuIRN')
+to_field 'cho_subject', column('iconography')
+to_field 'cho_medium', column('material', split: '|')
+to_field 'cho_extent', column('measurement_height')
+to_field 'cho_extent', column('measurement_length')
+to_field 'cho_extent', column('measurement_outside_diameter')
+to_field 'cho_extent', column('measurement_tickness')
```

Comment:
I assume `tickness` is how it is in the data? Should we have a ticket to follow up with the data provider?

Reply:
I copy and pasted from the spreadsheet. I'll check the data.

Reply:
This is in fact in the data. Issue opened.